apps/src/code-studio/components/progress/teacherPanel/TeacherPanel.jsx

```diff
@@ -121,7 +121,10 @@ class TeacherPanel extends React.Component {
       viewAs === ViewType.Teacher && currentStudent;
 
     const displayLevelExamples =
-      viewAs === ViewType.Teacher && sectionData && sectionData.level_examples;
+      viewAs === ViewType.Teacher &&
+      sectionData &&
+      sectionData.level_examples &&
+      sectionData.level_examples.length > 0;
```

Comment:
I think this would be a good place to use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Reply:
TIL!

Reply:
I did not know that optional chaining existed in JS! 🤯

Reply:
Charlie added it a few months ago and I'm super excited about it! https://github.com/code-dot-org/code-dot-org/pull/39543